src/Redux/APIFunctions/hotels.js

```diff
@@ -0,0 +1 @@
+// Fetch data from the backend API
```

Comment:
- [ ] It seems this file is not being used.  Kindly delete it if this is the case.